### PR TITLE
HWPV-199: Update contact details page

### PR DESCRIPTION
**Pull request by ashleygyngell**

### CODE REVIEW

File: app/assets/sass/application.scss [hunk 1]
```diff
@@ -62,6 +62,10 @@ table.no-pad-right td {
     margin-top: 40px;
 }
 
+.table-short-column {
+    width: 200px !important;
```
Comment (tpmcgowan) [inline app/assets/sass/application.scss:66]: Ideally best to try and avoid using `!important` but can sometimes be least worst option when trying to work with a codebase like this...

**Review by tpmcgowan — DISMISSED** [review]
LGTM

File: app/assets/sass/application.scss (hunk 1 above)
Comment (ashleygyngell) [reply app/assets/sass/application.scss]: Thanks @tpmcgowan. I think this was there whilst i was trying to override something else. Turns out its not needed so have removed! 😄

**Review by simonnebMOJ — APPROVED** [review]
LGTM

**Pull request merged**